packages/reporter/src/lib/tag.cy.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import Tag from './tag'
+
+describe('Tag', () => {
+  const aliases = [
+    'route',
+    'agent',
+    'primitive',
+    'dom',
+  ]
+
+  const statuses = [
+    'passed',
+    'failed',
+  ]
+
+  const MockCommandContainer = ({ children }) => (
+    <div style={{ backgroundColor: '#171926', minHeight: '20px' }}>
```

Comment:
I find myself doing this "wrapper parent" pattern a LOT for styling in component tests. Not directly related to this PR, but it sure would be nice to include something out of the box for this (potentially) or just recommend this pattern.

Reply:
Yeah, I was doing this to 1) force styles similar to the parent it would be rendered in and 2) providing the wrapper like:
```
cy.mount(<div><Component/></div>) 
```
didn't render as a named component in the reporter logs for cy.mount() and it wasn't hard to read.